forgit.plugin.zsh

```diff
@@ -1,4 +1,5 @@
 # MIT (c) Wenxuan Zhang
+hash fzf > /dev/null 2>&1 || {echo "FZF not found and is requried for forgit" && return 1}
```

Comment:
Is it better to put this line below the 6th line? Then we can use `forgit::warn` to print the warning message:

``` bash
hash fzf &>/dev/null || {forgit::warn "FZF not found and is requried for forgit" && return 1}
```